XamarinCommunityToolkit/Views/AvatarView.shared.cs

```diff
@@ -191,10 +191,11 @@ void OnValuePropertyChanged()
 			Image.BatchCommit();
 
 			Label.BatchBegin();
+			Label.IsVisible = !Image.IsVisible;
```

Comment:
#441 Hide label if image is visible

Reply:
I didn't hide the Label when the Image is visible because I wanted to make text visible while the image is loading.
Now if the image has a source, the text will be empty while the source is loading (even if the URI/Path is broken)

Reply:
we should find a way to subscribe on "OnImageLoaded" and hide label in this case. it doesn't look good now:
![image](https://user-images.githubusercontent.com/33021114/95795303-b5675880-0cf2-11eb-80b7-84228934c365.png)

also adding Aspect property would be nice. what do you think?

Reply:
I totally agree with both points 👍

Reply:
What will happen if we just set the same background color to Image as we set to the entire control?
Maybe it will fix the initial issue?

Reply:
Good idea, I will try it!

Reply:
Ah, no, I think it won't work.. :( we still showing the image above the text.
So, yeah. Need to get the info about successful image load somehow

Reply:
Here is the custom image source handler

https://github.com/xamarin/XamarinCommunityToolkit/tree/main/XamarinCommunityToolkit/Views/GravatarImageSource

But probably that's too complex.

Reply:
As far as I read Image has property IsLoading which should cover over scenarios

Reply:
Yes, but it didn't provide info about loading status (success/failure)